nimble/host/src/ble_l2cap_coc.c

```diff
@@ -156,6 +156,21 @@ ble_l2cap_coc_srv_find(uint16_t psm)
     return srv;
 }
 
+int
+ble_l2cap_coc_server_mtu_update(uint16_t psm, uint16_t new_mtu)
```

Comment:
who calls this function?

Reply:
In the app layer, when L2CAP_MTU forth word is called.

```
FORTH_WORD(L2CAP_MTU, "L2CAP_MTU", (mtu -- ), "Set l2cap mtu")
{
    uint32_t mtu = POP(uint32_t);
    cble_ctx_t *cble = cble_get_ctx();
    
    if(mtu > MYNEWT_VAL(BLE_L2CAP_COC_MPS)){
        CILOG(ERROR, BLE, "L2CAP CoC MTU can't exceed max MPS: %dB", MYNEWT_VAL(BLE_L2CAP_COC_MPS));
        return FORTH_ERR;
    }
    NIMBLE_ERROR(ble_l2cap_coc_server_mtu_update(CBLE_L2CAP_PSM, mtu));
    CILOG(INFO, BLE, "L2CAP CoC MTU overriden to %ldB was %ldB", mtu, cble->l2cap_coc_mps);
    cble->l2cap_coc_mps = mtu;
    return FORTH_OK;
}